app/graphql/types/query_type.rb

```diff
@@ -0,0 +1,97 @@
+module Types
+  class QueryType < Types::BaseObject
+    graphql_name 'Query'
+    description 'The query root for this schema'
+
+    field :node, field: GraphQL::Relay::Node.field, null: true
+    # field :nodes, field: GraphQL::Relay::Node.plural_field, null: true
```

Comment:
delete?

Reply:
This was carried over from the previous type definition. #131